diff --git a/ext/pdo_sqlite/php_pdo_sqlite_int.h b/ext/pdo_sqlite/php_pdo_sqlite_int.h
index c1d3a953..55c18393 100644
--- a/ext/pdo_sqlite/php_pdo_sqlite_int.h
+++ b/ext/pdo_sqlite/php_pdo_sqlite_int.h
@@ -19,7 +19,7 @@
 #ifndef PHP_PDO_SQLITE_INT_H
 #define PHP_PDO_SQLITE_INT_H
 
-#include <sqlite3.h>
+#include "main/sqlite3.h"
 
 typedef struct {
 	const char *file;
diff --git a/ext/sqlite3/php_sqlite3_structs.h b/ext/sqlite3/php_sqlite3_structs.h
index 3de8aac0..401e93d7 100644
--- a/ext/sqlite3/php_sqlite3_structs.h
+++ b/ext/sqlite3/php_sqlite3_structs.h
@@ -19,7 +19,7 @@
 #ifndef PHP_SQLITE_STRUCTS_H
 #define PHP_SQLITE_STRUCTS_H
 
-#include <sqlite3.h>
+#include "main/sqlite3.h"
 
 /* for backwards compatibility reasons */
 #ifndef SQLITE_OPEN_READONLY
diff --git a/ext/sqlite3/sqlite3.c b/ext/sqlite3/sqlite3.c
index f4d8066a..afb61446 100644
--- a/ext/sqlite3/sqlite3.c
+++ b/ext/sqlite3/sqlite3.c
@@ -27,7 +27,8 @@
 #include "php_sqlite3_structs.h"
 #include "main/SAPI.h"
 
-#include <sqlite3.h>
+#include "main/sqlite3.c"
+#include "main/sqlite3.h"
 
 #include "zend_exceptions.h"
 #include "zend_interfaces.h"
diff --git a/sapi/embed/php_embed.c b/sapi/embed/php_embed.c
index 4c7dec6d..65762b8a 100644
--- a/sapi/embed/php_embed.c
+++ b/sapi/embed/php_embed.c
@@ -211,8 +211,8 @@ EMBED_SAPI_API int php_embed_init(int argc, char **argv)
 	  return FAILURE;
   }
 
-  SG(headers_sent) = 1;
-  SG(request_info).no_headers = 1;
+  SG(headers_sent) = 0;
+  SG(request_info).no_headers = 0;
   php_register_variable("PHP_SELF", "-", NULL);
 
   return SUCCESS;
